### PR TITLE
fix: add Anurag to the list

### PR DESCRIPTION
**Pull request by kr-anurag**
added my profile to linkfree :yeye:

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/721"></a>

### CODE REVIEW

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
Please can you remove the package.json from your commits. @kr-anurag thank you

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

**Comment by kr-anurag** [comment]
ah hey, i will do the changes shortly, thanks

**Comment by kr-anurag** [comment]
hey, i have updated the changes

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
I still see the `package-lock.json` file being modified. Please update accordingly.

**Comment by kr-anurag** [comment]
ah hey @Panquesito7 
apparently it all didn't worked as expected i guess, an i close this PR? and make a new one?

File: public/data/anurag.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://twitter.com/kr_anurag_",
+        "icon": "twitter"
+      },
+      {
+          "name": "Join us on Discord",
+          "url": "https://discord.gg/MqsS6axMrZ",
+          "icon": "discord"
+      }
+    ]
+  }
```
Comment (krishguptadev) [inline public/data/anurag.json:22]:
```suggestion
}

```

**Comment by Panquesito7** [comment]
> ah hey @Panquesito7 apparently it all didn't worked as expected i guess, an i close this PR? and make a new one?

Best to update in the current PR. 😄

**Review by krishguptadev — CHANGES_REQUESTED** [review]
@kr-anurag you can run `git reset package-lock.json origin/main` to fix this 😀

**Comment by kr-anurag** [comment]
ah hey, lemme do it. Thanks!

**Comment by kr-anurag** [comment]
well should i revert the commit first, or just run it as it is, like it didnt worked even now

File: public/data/anurag.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://github.com/kr-anurag",
+        "icon": "github"
+      },
+      {
+        "name": "Connect with me on Twitter",
+        "url": "https://twitter.com/kr_anurag_",
+        "icon": "twitter"
+      },
+      {
+          "name": "Join us on Discord",
```
Comment (Panquesito7) [inline public/data/anurag.json:17]:
```suggestion
          "name": "Join my Discord community",
```

File: public/data/anurag.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Anurag",
+    "bio": "Teen | Frontend Developer | Student",
```
Comment (Panquesito7) [inline public/data/anurag.json:3]:
```suggestion
    "bio": "Teen | Front-end Developer | Student",
```

**Comment by kr-anurag** [comment]
well hey @Panquesito7, I think I had to pull commits from the main repo, and it was after when I added my profile, so it resulted in 8 commits, and now I'm confused about where to run `git reset` command, i think I might have to close this PR? or lemme fix it... thanks!

**Comment by kr-anurag** [comment]
hey, im sorry, but i will have to close this PR, and i will make a new one, apparently, nothing worked as expected

File: public/data/anurag.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://twitter.com/kr_anurag_",
+        "icon": "twitter"
+      },
+      {
+          "name": "Join my Discord community",
+          "url": "https://discord.gg/MqsS6axMrZ",
+          "icon": "discord"
+      }
+    ]
+  }
```
Comment (krishguptadev) [inline public/data/anurag.json:22]:
```suggestion
}

```

Comment (kr-anurag) [reply public/data/anurag.json:22]: apparently doing this won't solve the issue, what should I do now? like can I make a new PR?

Comment (krishguptadev) [reply public/data/anurag.json:22]: Sure, but best to this here itself.

Comment (kr-anurag) [reply public/data/anurag.json:22]: actually, there have been  9 commits now, by pulling and all that stuff, and I'm really sorry, I don't think I can solve it here itself, sorry! But I will make a new one

Comment (krishguptadev) [reply public/data/anurag.json:22]: Sure, feel free to close this PR and open a new one 😄

**Pull request closed without merging**